src/lib/mitt/index.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Since this file is copied from `next`, should we leave a note here?  Like `// copied from next.js/src/.../mitt.js`?

Reply:
I think that makes sense, probably even by commit permalink. You COULD import it from a deep import but of course it's subject to break in the future since it's internal.

Reply:
I was thinking the same thing!  Import directly from `next/src...`, but that would create a fragile dependency.  Too bad we can't just import `mitt`!